common/lib/dependabot/file_fetchers/base.rb

```diff
@@ -61,10 +66,17 @@ def target_branch
         source.branch
       end
 
+      # Fetches the language-specific dependency files for
+      # the repo this instance was created with.
+      # Returns an array of `Dependabot::DependencyFile` instances.
       def files
         @files ||= fetch_files
       end
 
+      # Returns the commit SHA-1 hash at the time
+      # the dependency files were fetched.
+      # If called before files, the returned value
+      # will be used in subsequent calls to files.
```

Comment:
@mattt Do you want to pull these docstrings out into a separate PR and add them? They look useful...